src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationModal.js

```diff
@@ -26,35 +26,48 @@ import { ButtonPrimary, ButtonSecondary } from '../../../generic/buttons'
 
 const prioritizeConfirmedAnnotations = (a, b) => b.is_confirmed - a.is_confirmed
 
-const ImageAnnotationModal = ({ imageId, setImageId }) => {
+const ImageAnnotationModal = ({ imageId, setImageId, benthicAttributes, growthForms }) => {
   const { databaseSwitchboardInstance } = useDatabaseSwitchboardInstance()
   const { projectId } = useParams()
   const [dataToReview, setDataToReview] = useState()
-  const [growthForms, setGrowthForms] = useState()
-  const [benthicAttributes, setBenthicAttributes] = useState()
   const [highlightedAttributeId, setHighlightedAttributeId] = useState('')
 
+  const getBenthicAttributeLabel = (benthicAttributeId) => {
+    const matchingBenthicAttribute = benthicAttributes.find(({ id }) => id === benthicAttributeId)
+    return matchingBenthicAttribute?.name ?? ''
+  }
+
+  const getGrowthFormLabel = (growthFormId) => {
+    const matchingGrowthForm = growthForms.find(({ id }) => id === growthFormId)
+    return matchingGrowthForm?.name.toLowerCase() ?? ''
+  }
+
+  const getAttributeGrowthFormLabel = ({ benthic_attribute, growth_form }) =>
+    growth_form
+      ? `${getBenthicAttributeLabel(benthic_attribute)} ${getGrowthFormLabel(growth_form)}`
+      : getBenthicAttributeLabel(benthic_attribute)
+
   const _fetchImageAnnotations = useEffect(() => {
     if (databaseSwitchboardInstance && projectId) {
       databaseSwitchboardInstance
         .getAnnotationsForImage(projectId, imageId)
         .then((data) => {
-          data.points.map((point) => point.annotations.sort(prioritizeConfirmedAnnotations))
-          setDataToReview(data)
+          const formattedPoints = data.points.map((point) => {
```

Comment:
hmm we both have ways of formatting (in my upcoming table updates pr). might be good to go over it together